script/script.js

```diff
@@ -82,28 +86,80 @@ function removeThis(e){
 }
 
 
-function storeIdea (id, card) {
-	localStorage.setItem(id, JSON.stringify(card));
-}
 
 $('.main-title , .idea-input').on('keyup', checkValue); 
 
-function checkValue(e) {
-	if (e.keyCode === 13 && ($('.main-title').val() !== '' && $('.idea-input').val() !== '')){
+function checkValue(e) {;
+	if (e.keyCode === 13 && $('.main-title').val() !== '' && $('.idea-input').val() !== ''){
+		e.preventDefault();
 		addIdea();
 	}
-};
+}
 
 $('.bookmark-list').on('click', '.upvote-button', upVoteTodo);
 
 function upVoteTodo(){
-	var qualityArr = ['swill', 'plausible', 'genius'];
-	var quality = $(this).parent().find('.quality-content').text();
-    quality > qualityArr[0] ? quality = qualityArr[1] : quality = qualityArr[2]
-    $(this).parent().find('.quality-content').text(quality);
-	console.log(quality)
+	var currentId = ($(this).closest('.idea-article').attr('id'));
+	var currentQuality = ($(this).parent().find('.quality-content').text());
+	var qualityValue = retQualityValue(currentQuality);
+	if (qualityValue === 1){
+	}else{
+		qualityValue--;
+	}
+	var newQuality = retrieveQuality(qualityValue);
+	($(this).parent().find('.quality-content').text(newQuality));
+	resetCard(newQuality,currentId);
+}
+
+$('.bookmark-list').on('click', '.downvote-button', downVoteTodo);
+
+function downVoteTodo(){
+	var currentId = ($(this).closest('.idea-article').attr('id'));
+	var currentQuality = ($(this).parent().find('.quality-content').text());
+	var qualityValue = retQualityValue(currentQuality);
+	if (qualityValue === 5){
+	}else{
+		qualityValue++;
+	}
+	var newQuality = retrieveQuality(qualityValue);
+	($(this).parent().find('.quality-content').text(newQuality));
+	resetCard(newQuality,currentId);
+}
+
+function retQualityValue(current){
```

Comment:
These 2 functions are nearly identical, it feels like we could reuse one with a bit of a tweak.